frontend-react/e2e/spec/all/managing-your-connection-page.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { scrollToFooter, scrollToTop } from "../../helpers/utils";
+import { ManagingYourConnectionPage } from "../../pages/managing-your-connection";
+import { test as baseTest, expect } from "../../test";
+
+const cards = [
+    {
+        name: "For healthcare organizations",
+        links: [
+            "Manage your public key",
+            "View your submission history",
+            "Login",
+            "contact us",
+        ],
+    },
+    {
+        name: "For public health agencies",
+        links: [
+            "Refer healthcare organizations",
+            "View your dashboard",
+            "Login",
+            "contact us",
+        ],
+    },
+];
+
+export interface ManagingYourConnectionPageFixtures {
+    managingYourConnectionPage: ManagingYourConnectionPage;
+}
+
+const test = baseTest.extend<ManagingYourConnectionPageFixtures>({
+    managingYourConnectionPage: async (
+        {
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        },
+        use,
+    ) => {
+        const page = new ManagingYourConnectionPage({
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        });
+        await page.goto();
+        await use(page);
+    },
+});
+
+test.describe(
+    "Managing Your Connection page",
+    {
+        tag: "@smoke",
+    },
+    () => {
+        test("has correct title", async ({ managingYourConnectionPage }) => {
+            await expect(managingYourConnectionPage.page).toHaveTitle(
+                managingYourConnectionPage.title,
+            );
+            await expect(managingYourConnectionPage.heading).toBeVisible();
+        });
+
+        test.describe("Quick links", () => {
+            for (const card of cards) {
+                // eslint-disable-next-line playwright/expect-expect
```

Comment:
We've implicitly agreed that we're ok with conditional tests, ie test within loops, we should probably just update our eslint config?